mosaic/README.md

```diff
@@ -0,0 +1,75 @@
+# Mosaic
+
+Mosaic is a tool that allows you to display images in your terminal programs. It
```

Comment:
Just a suggestion...

```suggestion
Mosaic is a tool that allows you to display images in your terminal emulators. It
```

Reply:
I'll keep it as terminal programs as it's a library. If we made it a standalone CLI app, then I would say it's a tool that allows you to display images in your terminal emulator 🙏